pkg/controller/controller.go

```diff
@@ -1370,6 +1479,144 @@ func (c *controller) bindingDelete(obj interface{}) {
 	glog.V(4).Infof("Received delete event for Binding %v/%v", binding.Namespace, binding.Name)
 }
 
+// getServiceClassPlanAndBroker is a sequence of operations that's done in couple of
```

Comment:
This comment seems unnecessarily wordy; I feel like the common use case for a function in general is for "a sequence of operations that's done in a couple of places". I'd just put something like:

```
getServiceClassPlanAndBroker returns the class, plan, broker name, and a newly-constructed broker client for a given instance.
```